modules/rational-org.el

```diff
@@ -0,0 +1,18 @@
+;;; rational-org.el  -*- lexical-binding: t; -*-
+
+(straight-use-package 'org)
```

Comment:
Great starting point!

Reply:
I wonder if it will not be better to use the default `org` package that comes with Emacs? If any other package requires a newer version, then it will be updated (as with `org-roam`).

Reply:
I mean: it would be "rational" to use what ever emacs comes with, don't you think? Is this not the general approach of this project?